run

```diff
@@ -270,6 +292,28 @@ def build_swift_toolchain(workspace, args):
         raise common.UnsupportedPlatform
     common.check_execute(build_command, timeout=9999999)
 
+def build_clang(workspace, source_path):
+    build_path = os.path.join(workspace, 'build_clang_source_compat')
+    source_path = os.path.join(source_path, 'llvm')
+    common.check_execute(['mkdir', '-p', build_path])
+
+    with common.DirectoryContext(build_path):
+        try:
+            ninja_path = common.check_execute_output(['xcrun', '--find', 'ninja']).strip().decode("utf-8")
+            cmake_command = [
+                'xcrun', 'cmake', '-G', 'Ninja',
+                '-DCMAKE_MAKE_PROGRAM={}'.format(ninja_path),
+                '-DLLVM_ENABLE_PROJECTS=clang;llvm',
+                '-DCMAKE_BUILD_TYPE=Release',
+                '-DCLANG_APPLE_BUILD_VERSION_STRING=13000000',
```

Comment:
I suppose you are hardcoding this version number because it’s the current clang version and it’s bound to get out of date at some point. I’d prefer something completely non-sensical like `99999999` or to remove it, if possible. Otherwise we’ll wonder in a couple of years why this needs to be `13000000`.

Reply:
That specific string is necessary for building an internal project, but I removed it from this PR.